package.json

```diff
@@ -80,6 +80,7 @@
         "pg": "^7.12.1",
         "pino": "^5.13.5",
         "typeorm": "0.2.18",
+        "web3-providers": "^2.0.0-alpha.1",
```

Comment:
This dep is unused.

Reply:
@xianny I think I got an error that this wasn't installed too. `rm -rf node_modules` fixed it.

Reply:
I was attempting to import the JsonRpcResponseValidator from here but that didn't work. Will remove this line.